isb_lib/geome_adapter.py

```diff
@@ -323,7 +338,7 @@ def reparseThing(thing):
     return thing
 
 
-def loadThing(identifier: str, t_created: datetime.datetime, existing_thing: Optional[Thing]) -> Thing:
+def loadThing(identifier: str, t_created: datetime.datetime, existing_thing: Optional[Thing], ids: GEOMEIdentifierIterator) -> Thing:
```

Comment:
New parameter so that when we load things we have the necessary context to look up the `local_contexts_id` for the project.